### ISSUE TITLE
I3D model performance on Kinetics 400 

### ISSUE THREAD

**Issue by busixingxing** [open-mmlab/mmaction#119]
Hi,

I am doing some training and testing using a pre-trimed version of Kinetics-400, because youtube is not letting crawlers to do mass scale download anymore. 

The data download link is here. [dropbox link](https://www.dropbox.com/s/wcs01mlqdgtq4gn/compress.tar.gz?dl=1)

When I tried to use this version of dataset and VideoDataset pipeline (use decord) to train an I3D model, I increase the  [image per gpu](https://github.com/open-mmlab/mmaction/blob/bb31acb5a060e351ba4261d94558fdf7f6bdfb5d/configs/kinetics400/i3d_kinetics400_3d_rgb_r50_c3d_inflate3x1x1_seg1_f32s2_video.py#L47) to 16 and only used 4 gpus. The training takes one week. 

If I use validation data to do the testing, the accuracy is 

> Mean Class Accuracy = 54.34
    Top-1 Accuracy = 53.97
    Top-5 Accuracy = 76.45

Then I tried to use the pre-trained model in mmaction model zoo's [checkpoint](https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmaction/models/kinetics400/i3d_kinetics_rgb_r50_c3d_inflated3x1x1_seg1_f32s2_f32s2-b93cc877.pth) and got:

> Mean Class Accuracy = 35.41
Top-1 Accuracy = 34.89
Top-5 Accuracy = 57.78

I am wondering if anyone can repeat the result also using this version of kenetics-400. Thanks. 

BTW, the dataset I downloaded, have a few video that is too short for training, so I dropped all the video that have less than 85 frames.

**Comment by zhaoyue-zephyrus** [comment]
Hi @busixingxing 

Thanks for reaching out.

The data that you used (according to the dropbox link) preprocesses each video such that the shorter edge is 256 while keeping the original aspect ratio. According to our implementation, the videos are rescaled to a fixed size of 340x256. I think this may account for most of the disprepency. What I recommend is that you processess that val data into our format (340x256) and do the inference to check if it works. Then you may continue to train yourself.

Btw, we are about to release the newer model which use the data format like FAIR does (ie. Shorter edge scaled to 256). You can try that as well.

**Comment by busixingxing** [comment]
@zhaoyue-zephyrus Thank you so much for the response. Look forward to try your new pre-trained model.

**Comment by zhaoyue-zephyrus** [comment]
@busixingxing 

The new pretrained model has been uploaded. Please refer to [MODEL_ZOO](https://github.com/open-mmlab/mmaction/blob/master/MODEL_ZOO.md).

**Comment by busixingxing** [comment]
Thank you @zhaoyue-zephyrus
